IntoHistory/HeroListCell.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  HeroListCell.swift
+//  IntoHistory
+//
+//  Created by 김민택 on 2022/08/12.
+//
+
+import UIKit
+
+class HeroListCell: UICollectionViewCell {
+    
+    // MARK: - Property
+    
+    static let identifier = "HeroListCell"
+    
+    // MARK: - View
+    
+    let imageView: UIImageView = {
+        return $0
+    }(UIImageView())
```

Comment:
p2;

현재 미뉴가 작성해주신 코드를 아이폰6s에서 빌드했을 때, 무궁화 동그라미가 조금 길쭉하게 비율이 망가져서 나오게 되는데요,,! 그래서 ```scaleAspectFit``` 을 적용해주면 좋을 것 같습니다.

```suggestion
    let imageView: UIImageView = {
        $0.contentMode = .scaleAspectFit
        return $0
    }(UIImageView())
```

|아이폰 6s에서 무궁화가 찌그러지는 화면| 변경 후 |
|---|---|
|![simulator_screenshot_3D3321A0-4BD6-4975-B8C7-21C9BC7B3686](https://user-images.githubusercontent.com/96969693/184476573-5a9ebf9e-d38f-4ef8-a58d-411767e91137.png)|![simulator_screenshot_0FFAE47C-C153-4EB5-9408-A5F27AFA88E8](https://user-images.githubusercontent.com/96969693/184476641-35cfae3a-6981-49ab-a4a9-75263cd4daf8.png)|

Reply:
아 그리고 글씨가 잘리는 이슈도 있는데, 이는 제가 알맞는 suggestion 코드를 못찾겠네요 ㅠㅠ 찾는대로 바로 댓글달겠습니다!~!